vip-integrations.php

```diff
@@ -14,16 +14,22 @@
 
 defined( 'ABSPATH' ) || die();
 
+// @codeCoverageIgnoreStart - the actual code here is tested individually in the unit tests.
```

Comment:
I ended up adding this here as this is where the registrations happen for the actual plugins. The underlying logic with the real plugins is better tested with the fake-integration that's in the unit tests. Instead, I added two tests for two of the integrations that didn't have unit tests and attempted to add what I could in there.

So for next time if an integration is added, what would be done is:

- Add the integration here
- Add the integration under integrations, and then add a test for it as well.

So that way as much of the new code is covered via the unit test.

Hope that makes sense, and makes it clear why I did this.